pkg/server/server_test.go

```diff
@@ -217,6 +226,71 @@ var _ = Describe("Applications handler", func() {
 
 		Expect(r.Applications).To(HaveLen(1))
 	})
+
+	It("get commits", func() {
+		log := makeFakeLogr()
+		kubeClient := &kubefakes.FakeKube{}
+		kubeClient.GetApplicationStub = func(context.Context, types.NamespacedName) (*wego.Application, error) {
+			return &wego.Application{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "my-app",
+					Namespace: "wego-system",
+				},
+				Spec: wego.ApplicationSpec{
+					Branch: "main",
+					Path:   "./k8s",
+					URL:    "github.com/test",
+				},
+			}, nil
+		}
+
+		appSrv := app.New(logger.NewCLILogger(os.Stderr), nil, nil, kubeClient, nil)
+		gitProviders := &gitprovidersfakes.FakeGitProvider{}
+		var commits []gitprovider.Commit
```

Comment:
```suggestion
		commits := []gitprovider.Commit{commitInfo: gitprovider.CommitInfo{Author: "testAuthor"}}
```

No need to use `append` below if you are going to declare a literal. I (typically) only use `append` in a loop.